materials/functions_R.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: page
+element: notes
+title: Functions
+language: R
+---
+
+### Understandable and reusable code
+
+* Write code in understandable chunks.
+    * Clear variable names
+    * Consistent indentation and line spacing
+    * Good commenting
+    * Functions
+* Write reusable code.
+    * Concise and modular script
+    * Functions with general structure 
+
+### Understandable chunks
+
+* Human brain can only hold ~7 things in memory.
+    * Write programs with no more than ~7 things.
```

Comment:
"Write programs..." -> "Write programs that don't require remembering more than ~7 things at once."